src/components/Resources/Schema/Create.tsx

```diff
@@ -94,7 +94,11 @@ const CreateSchema = () => {
 
 	const areAllInputsFilled = (formData: IFormData) => {
```

Comment:
Think we should change the function name

Reply:
Resolved.